modules/renter/files.go

```diff
@@ -239,6 +264,11 @@ func (r *Renter) FileList() []modules.FileInfo {
 		offline := r.hostContractor.IsOffline(id)
 		return offline
 	}
+	goodForRenew := func(id types.FileContractID) bool {
+		id = r.hostContractor.ResolveID(id)
+		cu, ok := r.hostContractor.ContractUtility(id)
```

Comment:
should we expand the `cu` object to also include the `IsOffline` information? That will limit the number of times that we are calling out of package